core/pkg/evaluator/json.go

```diff
@@ -150,8 +150,12 @@ func (je *Resolver) ResolveAllValues(ctx context.Context, reqID string, context
 	_, span := je.tracer.Start(ctx, "resolveAll")
 	defer span.End()
 
-	var err error
-	allFlags, flagSetMetadata, err := je.store.GetAll(ctx)
+	selector := store.NewSelector("")
+	s := ctx.Value(store.SelectorContextKey{})
+	if s != nil {
+		selector = s.(store.Selector)
+	}
+	allFlags, flagSetMetadata, err := je.store.GetAll(ctx, &selector, nil)
```

Comment:
The `IStore` interface (and implementation) is also used by the in-process provider in the go-sdk-contribs. The in-process provider will not need to support selectors/quering as flagd does, so the interface was not modified. The selector is instead propagated in the `ctx`, and if it's set, used in the storage layer.